features/step_definitions/deleting_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given(/^there is users named:$/) do |table|
+  table.hashes.each do |hash|
+    FactoryGirl.create(:user, hash)
+  end
+end
+
+Given(/^I send an email to "([^"]*)"$/) do |name|
```

Comment:
This is not a blocker, but there are helper methods in the "tips and tricks" section that you could use instead of going through the site in your test. See if you can't refactor this to use the "I send a message" helper.